packages/shared/lib/services/connection.service.ts

```diff
@@ -1742,6 +1742,90 @@ class ConnectionService {
         return Err(new NangoError('failed_to_get_connections_count'));
     }
 
+    async billableConnections(referenceDate: Date): Promise<
+        Result<
+            {
+                accountId: number;
+                count: number;
+                year: number;
+                month: number;
+            }[],
+            NangoError
+        >
+    > {
+        // Note:
+        // a billable connection is a connection that is not deleted and has not been deleted during the month
+        // connections are pro-rated based on the number of seconds they were existing in the month
+
+        const targetDate = new Date(Date.UTC(referenceDate.getUTCFullYear(), referenceDate.getUTCMonth(), referenceDate.getUTCDate(), 0, 0, 0, 0));
```

Comment:
went with setting billing start/end of month to be UTC. I guess I some point we are gonna want to be account specific but I don't think we have an account timezone setting yet and unless there is any legal requirement using UTC is simpler